### ISSUE TITLE
Desktop: Screen reader accessibility: Deleting a note announces raw HTML

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#11742]
See [this forum post](https://discourse.joplinapp.org/t/calling-visually-impaired-users-help-us-improve-accessibility/42094/7?u=personalizedrefriger):
> There's a region where status messages and such appear but it's a bit confusing. For instance, here's what's there right now for me and has been for the last several minutes (note the raw HTML):
> ```
> The note was successfully moved to the trash. <a href="#" class="cancel" data-lastDeletion="{&quot;noteIds&quot;:[&quot;3a228337c9f94d58a92b113b13eafba1&quot;],&quot;folderIds&quot;:[],&quot;timestamp&quot;:1738004505109}" id="deletion-notification-cancel-447022">Cancel</a>
> ```
>
> Questions about this specifically:
>
> -  I'm guessing this should be some sort of actionable notification area? If so then it should be a region too.
> - Do these notifications stick around, and are there keyboard commands to dismiss/navigat them?

**Note**: This might require forking/patching the [Notyf library](https://github.com/caroso1222/notyf).

**Comment by personalizedrefrigerator** [comment]
Still an issue! (See the linked pull request).

**Comment by personalizedrefrigerator** [comment]
Still an issue (see linked PR).
